Ladybird/Qt/SettingsDialog.cpp

```diff
@@ -114,18 +121,4 @@ void SettingsDialog::setup_search_engines()
     });
 }
 
-void SettingsDialog::closeEvent(QCloseEvent* event)
-{
-    save();
-    event->accept();
-}
-
-void SettingsDialog::save()
-{
-    auto url_string = MUST(ak_string_from_qstring(m_new_tab_page->text()));
-    if (!URL(url_string).is_valid())
-        return;
-    Settings::the()->set_new_tab_page(m_new_tab_page->text());
```

Comment:
I'm still not convinced that this is the correct UX for the settings page. In SerenityOS we have a pattern of a `[ Ok ] [ Cancel ] [ Apply ]` type thing for Settings. Is there precedence for this kind of "save as you go" UX for macOS and Linux apps?